Packs/OktaITAdmin/TestPlaybooks/playbook-OktaITAdmin-Test.yml

```diff
@@ -0,0 +1,655 @@
+id: OktaITAdmin-Test
```

Comment:
I don't remember if we do have, but please move to NonCircleTests if we don't have an instance, otherwise remove the "No tests" in the integration yml

Reply:
Yes. Can you please check with Rashmi for instance/credential details.